### DISCUSSION TITLE
Inconsistency with the Oxygenite Tank equipped Diving Mask and Exosuit.

### DISCUSSION THREAD

**Discussion by Smrtonosan** [FakeFishGames/Barotrauma#12326] [Ideas and suggestions]
Oxygenite Tank fitted Exosuit does not benefit from a movement speed increase, however if you equip a Oxygenite Tank fitted Diving Mask under the Exosuit you will gain the movement speed increase. Which is odd seing as it's the same oxygenite, an argument could be made that it's because the Exosuit prolongs the tank's duration, but the Abyss Suit does the same while also granting the speed increase.

I don't know if this is:
a.) Intentional, you are supposed to choose between wearing a helmet for concussion protection or wearing a Diving Mask for speed.
b.) Oversight, you are not supposed to use Oxygenite Tanks to increase your movement speed in a Exosuit no matter what.

**Comment by G-M-twostay** [comment]
could it be that the speed gain is too small in exosuit that you barely notice it?(idk i'm just guessing)
